style.css

```diff
@@ -93,3 +93,394 @@ h6 {
+  font-family: 'Inter', sans-serif;
+  font-weight: bold;
+  font-size: 17px;
+  font-style: normal;
+  line-height: 24px;
+}
+
+.about-me {
+  background-color: #1c1a19;
+  width: 375px;
+  height: 600px;
+  background-image: url(/image/Group2.png);
+  background-repeat: no-repeat;
+  background-position: 100% 100%;
+}
+
+.line-1 {
+  height: 1px;
+  width: 300px;
+  background-color: #fff;
+  margin-left: 2rem;
+  margin-top: -5rem;
+}
+
+.ignore {
+  color: #1c1a19;
+}
+
+.about-section-h1 {
+  display: flex;
+  justify-content: center;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 70px;
+  line-height: 88px;
+  padding-top: 4rem;
+  padding-right: 1.6rem;
+  padding-left: 1.6rem;
+}
+
+.about-section-paragraph {
+  display: flex;
+  justify-content: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  padding: 0 2rem;
+}
+
+.about-section-button {
+  height: 48px;
+  background-color: #ff6b00;
+  color: #fff;
+  border: none;
+  margin-top: 1.5rem;
+  font-family: 'Inter', sans-serif;
+  font-weight: bold;
+  font-size: 17px;
+  font-style: normal;
+  line-height: 24px;
+  margin-left: 2rem;
+  padding: 1rem;
+}
+
+.aboutme-container2 {
+  background-color: #1c1a19;
+  width: 375px;
+  height: 1400px;
+  padding-top: 8rem;
+}
+
+.svg-language {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+#language {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 27px;
+  color: #fff;
+}
+
+.js-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  color: #fff;
+  padding-top: 2rem;
+}
+
+.btnn {
+  padding: 1rem 0;
+  color: #fff;
+}
+
+.svg2-language {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 3rem;
+}
+
+#framework {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 27px;
+  color: #fff;
+}
+
+.js-section2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 43px;
+  color: #fff;
+  padding-top: 2rem;
+}
+
+.svg3-language {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 3rem;
+}
+
+#skills-section {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 27px;
+  color: #fff;
+}
+
+.section3 {
+  margin-bottom: 4rem;
+}
+
+.js-section3 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 43px;
+  color: #fff;
+  padding-top: 2rem;
+}
```

Comment:
- [ ] I can still see that your project seems to be responsive only for one screen size `362px` but remember the mobile starts from `480px`, this is how your portfolio looks like on `480px` 👇 
![image](https://user-images.githubusercontent.com/55465133/129418751-b0a03c4a-ab0f-4d03-a8da-ec88a66aa755.png)
![image](https://user-images.githubusercontent.com/55465133/129418789-5893dd99-ffe3-40bb-a793-67e06dcfb4b3.png)

Please let's follow the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) as much as possible, I am sure you can do it 😉

Reply:
I have fixed this. Thank you